includes/class-wc-webhook.php

```diff
@@ -306,6 +307,31 @@ protected function is_already_processed( $arg ) {
 		return false !== array_search( $arg, $this->processed, true );
 	}
 
+	/**
+	 * Make retry enabled determination.
+	 *
+	 * @since 4.4.0
+	 * @param int $arg Resource ID
+	 * @param WP_Error|HTTP_Response $response
+	 * @return bool
+	 */
+	public function is_retryable( $arg, $response ) {
```

Comment:
What do you think of `can_retry` instead?

Reply:
Maybe `should_retry()`?